acceptance/suites/pre_suite/foss/30_install_dev_repos.rb

```diff
@@ -26,7 +26,7 @@
   end
 end
 
-confine_block(:to, {:platform => puppetdb_supported_platforms}, master) do
```

Comment:
@erikPrime Are we also going to have to change this line to do something similar - https://github.com/puppetlabs/puppetserver/blob/7875b01518a8c3c178eba20c46909b2e79ca09e6/acceptance/suites/tests/00_smoke/puppetdb_integration.rb#L13 - or is the problem with Beaker that is forcing us to do this related just to `confine_block` and not `confine`?

I guess if the problem is with `confine_block` and not `confine` and `confine` is a more conventional way to restrict the platforms that the script is run against than the `if` block, we could also just use `confine` and move this into its own Ruby file?  Not sure what's the best way to do this sort of thing...